src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/Example2.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MatchXpath">
+      <property name="query" value="//CTOR_DEF[count(./PARAMETERS/*)
+              > 0]"/>
```

Comment:
Please put value attribute to new line, to avoid line wrapping in `"`

Reply:
Please apply to all others.

Reply:
Even after moving to the new line the  for some of the examples value field needs line wrapping.

Reply:
ok, please minimize wrapping.

Reply:
Done.